src/handlers/feature_handlers.h

```diff
@@ -421,7 +421,7 @@ class vehSirenHandler {
 public:
     void Update();
     void Reset();
-    void SizeOf();
+    MM2::vehSiren* OperatorNew();
```

Comment:
You also need to fix calling convention (by making it static) and adding the size parameter

Reply:
Roger that